src/pages/workspaces/workspace/Files.ts

```diff
@@ -46,6 +50,22 @@ export const Files = _.flow(
         rootLabel,
         title: 'Files',
         onChangePath: setPath,
+        extraMenuItems: h(
+          Link,
+          {
+            href: `https://seqr.broadinstitute.org/workspace/${workspaceInfo.namespace}/${workspaceInfo.name}`,
+            style: { padding: '0.5rem' },
+            target: '_blank',
+            onClick: async () => {
+              await Ajax().Metrics.captureEvent(Events.workspaceFilesSeqr, {
+                workspaceNamespace: workspaceInfo.namespace,
+                workspaceName: workspaceInfo.name,
+                success: true,
+              });
```

Comment:
1. we typically treat Mixpanel events as fire-and-forget; no need for the user to wait for anything. I suggest removing the `await` from here.
2. Is the `success: true` useful here? It's hardcoded to a constant value; it won't provide any filtering capabilities in a mixpanel dashboard.